pages/community/precice-workshop-2024.md

```diff
@@ -31,20 +31,22 @@ The registration for the workshop will open in the next few weeks. You will be a
 
 #### Early-bird registration
 
-| Ticket type   | Members of academia | Members of industry  |
-|--------------|-----------|------------|
-| **full registration** | 300 Euro    | 500  Euro     |
-| **main part only**      | 200 Euro  | 300 Euro       |
+| Ticket type | Members of academia | Members of industry  |
+|---|---|---|
+| **workshop + course** | 300 Euro    | 500  Euro     |
+| **workshop**      | 200 Euro  | 300 Euro       |
```

Comment:
I find the extra bolds a bit too much visually, the table already conveys the meaning without extra styling.

General good practice: separate the style from the content as much as possible.

Reply:
extra bolds remove